benchmark_runner/common/ocp_resources/odf/template/01_delete_disks_template.sh

```diff
@@ -4,5 +4,6 @@ while (($# >= 2)) ; do
     node="$1"
     delete_disk_command="$2"
     oc debug "node/$node"  -- chroot /host sh -c "$delete_disk_command"
+    oc debug "node/$node" -- chroot /host sh -c "[ -d /mnt/local-storage/ ] && rm -rf /mnt/local-storage/ || echo 'Directory /mnt/local-storage/ does not exist'"
```

Comment:
Where does /mnt/local_storage come from?  Could it be passed in to the script as a third argument?

Reply:
Its hard coded path for ODF storage

Reply:
This is not need if you are doing a complete fresh install.

Reply:
Removed